app/controllers/saml_idp_controller.rb

```diff
@@ -17,6 +17,7 @@ class SamlIdpController < ApplicationController
   prepend_before_action :skip_session_expiration, only: [:metadata, :remotelogout]
 
   skip_before_action :verify_authenticity_token
+  before_action :require_path_year
```

Comment:
I forget

I think that it's basically only required in specs, because in controller tests you can route directly to an action and skip the routers constraints, but in real life the router handles them. 

I'll toy around with removing it and see what happens

Reply:
Confirmed, it was required because the controller specs let us route directly.